rdt/hyper_transformer.py

```diff
@@ -232,6 +242,39 @@ def update_default_sdtype_transformers(self, new_sdtype_transformers):
         """
         self.default_sdtype_transformers.update(new_sdtype_transformers)
 
+    def update_transformers(self, column_name_transformer):
+        """Update any of the transformers assigned to each of the column names.
+
+        Args:
+            column_name_transformer(dict):
+                Dict mapping column names to transformers to be used for that column.
+        """
+        if self._fitted:
+            warnings.warn(
+                "For this change to take effect, please refit your data using 'fit' "
+                "or 'fit_transform'."
+            )
+
+        if len(self.field_transformers) == 0:
+            self.print_tip(
+                'Use the `detect_initial_config` method to pre-populate all the sdtypes '
+                'and transformers from your dataset.'
+            )
+
+        for column_name, transformer in column_name_transformer.items():
+            current_sdtype = self.field_sdtypes.get(column_name)
+            if current_sdtype and current_sdtype != transformer.get_input_type():
```

Comment:
@npatki Does this method still run through if the user hasn't set the config or used `detect_initial_config`? If so, then should this error also raise if the `sdtype` currently in the config is `None`?

Reply:
+1 for consistency's sake, we can do the same error that we do when setting transformers by `sdtype`